package.json

```diff
@@ -142,12 +141,13 @@
     "mustache": "^2.2.1",
     "node-fetch": "^2.6.1",
     "papaparse": "^5.2.0",
+    "pmtiles": "^3.0.7",
```

Comment:
There is a pmtiles 4.0 released now, and that mentions removed pmtiles spec v2 support which significantly reduces bundle size. Is it worth upgrading to v 4.0, or does TerriaMap need pmtiles spec v2 support?